internal/cli/cmd/addon/addon.go

```diff
@@ -824,19 +824,25 @@ func addonListRun(o *list.ListOptions) error {
 				selectors = addon.Spec.Installable.GetSelectorsStrings()
 				autoInstall = addon.Spec.Installable.AutoInstall
 			}
+			label := obj.GetLabels()
+			provider := ""
+			if label != nil {
+				provider = label[constant.AddonProviderLabelKey]
+			}
```

Comment:
Get value from nil map is allowed, that will return empty, so checking nil is not necessary.